docs/_guide/templates.md

```diff
@@ -569,6 +581,162 @@ html`${until(content, html`<span>Loading...</span>`)}`
 
 For a list of directives supplied with lit-html, see [Built-in directives](https://lit-html.polymer-project.org/guide/template-reference#built-in-directives) in the Template syntax reference.
 
+## Accessing nodes in the shadow DOM
+
+Templated DOM is usually rendered into shadow DOM, so the nodes are not direct children of the component. Use `this.shadowRoot.querySelector()` or `this.shadowRoot.querySelectorAll()` to find nodes in the 
```

Comment:
maybe?

```suggestion
The `render()` method result is usually rendered into shadow DOM, so the nodes are not direct children of the component. Use `this.shadowRoot.querySelector()` or `this.shadowRoot.querySelectorAll()` to find nodes in the 
```

only because "templated DOM" seems to be a phrase of our own creation. I might change "templated DOM" below to "rendered DOM" too.

Reply:
It *is* a phrase of our own creation. We already use it a couple of times at the beginning of the chapter.

In the lit-html docs, we actually just say "templates"--like, "Styling templates"--when we really mean, "styling the DOM you create by rendering templates." If you think "rendered DOM" is clearer and less invented-y, we could rephrase this to, say, "Rendered DOM is usually placed in an open shadow root," or "The component's template is usually rendered into shadow DOM" and use "rendered DOM" below as you suggested.

Added this to today's docs meeting agenda if we need to discuss it further.